src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/HomeOfficeAppealOutcomePersonalisation.java

```diff
@@ -0,0 +1,75 @@
+package uk.gov.hmcts.reform.iacasenotificationsapi.domain.personalisation;
+
+import static java.util.Objects.requireNonNull;
+
+import com.google.common.collect.ImmutableMap;
+
+import java.util.Map;
+import javax.validation.constraints.NotNull;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.AsylumCase;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.AsylumCaseDefinition;
+
+@Service
+public class HomeOfficeAppealOutcomePersonalisation implements NotificationPersonalisation {
+
+    private final String appealOutcomeHomeOfficeTemplateId;
+    private final String allowedAppealHomeOfficeEmailAddress;
+    private final String dismissedAppealHomeOfficeEmailAddress;
+
+    public HomeOfficeAppealOutcomePersonalisation(
+        @Value("${allowedAppealHomeOfficeEmailAddress}") String allowedAppealHomeOfficeEmailAddress,
+        @Value("${dismissedAppealHomeOfficeEmailAddress}") String dismissedAppealHomeOfficeEmailAddress,
+        @NotNull(message = "appealOutcomeHomeOfficeTemplateId cannot be null") @Value("${govnotify.template.appealOutcomeHomeOfficeTemplateId}") String appealOutcomeHomeOfficeTemplateId) {
+
+        this.appealOutcomeHomeOfficeTemplateId = appealOutcomeHomeOfficeTemplateId;
+        this.allowedAppealHomeOfficeEmailAddress = allowedAppealHomeOfficeEmailAddress;
+        this.dismissedAppealHomeOfficeEmailAddress = dismissedAppealHomeOfficeEmailAddress;
+    }
+
+    @Override
+    public String getTemplateId() {
+        return appealOutcomeHomeOfficeTemplateId;
+    }
+
+    @Override
+    public String getEmailAddress(AsylumCase asylumCase) {
+
+        final String appealOutcomeDecision =
+            asylumCase.read(AsylumCaseDefinition.IS_DECISION_ALLOWED, String.class).orElse("");
+
+        final String appealOutcomeHomeOfficeEmailAddress;
+
+        if (!appealOutcomeDecision.equals("")) {
+
+            appealOutcomeHomeOfficeEmailAddress =
+                appealOutcomeDecision.equals("allowed")
+                    ? allowedAppealHomeOfficeEmailAddress : dismissedAppealHomeOfficeEmailAddress;
+        } else {
+            appealOutcomeHomeOfficeEmailAddress = "";
+        }
+
+        return appealOutcomeHomeOfficeEmailAddress;
```

Comment:
What happens when we have try to send to empty email later on with GovNotify api?
I think it would be better to throw exception as it is implemented in other Personalisation classes. When you can't find or when the property `IS_DECISION_ALLOWED` is empty for example `IllegalStateException`
What do you think?

Reply:
Thanks for the comments on this one, I appreciate it. Hopefully a better solution in place now.